scripts/PerfHarness/PerfHarness.cs

```diff
@@ -0,0 +1,34 @@
+using System.IO;
+using System.Reflection;
+using Microsoft.Xunit.Performance.Api;
+
+public class PerfHarness
+{
+    public static void Main(string[] args)
+    {
+        using (XunitPerformanceHarness harness = new XunitPerformanceHarness(args))
+        {
+            foreach(var testName in EnumeratePerfTests())
+            {
+                harness.RunBenchmarks(GetTestAssembly(testName));
+            }
+        }
+    }
+
+    private static string GetTestAssembly(string testName)
+    {
+        // Assume test assemblies are colocated/restored next to the PerfHarness.
+        return Path.Combine(
+            Path.GetDirectoryName(Assembly.GetEntryAssembly().Location),
+            $"{testName}.dll"
+        );
+    }
+
+    private static string[] EnumeratePerfTests()
+    {
+        return new [] {
```

Comment:
I assume longer term you will just discover new test assemblies.

For now, could you add a note to the instructions saying that this array needs to be modified to add new tests to new assembly.